homeassistant/components/mqtt/client.py

```diff
@@ -74,6 +73,7 @@
     # Only import for paho-mqtt type checking here, imports are done locally
     # because integrations should be able to optionally rely on MQTT.
     import paho.mqtt.client as mqtt
+    from paho.mqtt.client import MQTTMessage
```

Comment:
I suggest using two separate PR (one for mqtt and one for stream)
For mqtt, maybe replace `MQTTMessage` with `mqtt.MQTTMessage` to have only a single import.

Reply:
I still think it should be two separate PR (one for each component)